app/contrib/ds/admin_style/templates/admin/filer/widgets/admin_file.html

```diff
@@ -51,14 +51,16 @@
                 </div>
 
                 <div class="file-controls">
+                    {% if object.file.exists %}
                     <a href="{{ change_url }}?_edit_from_widget=1" class="js-related-edit related-lookup {% if object %}related-lookup-change{% endif %} edit" id="{{ id }}_change"
                         title="{% translate 'Edit' %}">
                         <span class="edit-file"><span class="fa fa-pencil cms-icon cms-icon-settings"></span></span>
                     </a>
+                    {% endif %}
 
                     <a href="{{ lookup_url }}" class="js-related-lookup related-lookup {% if object %}related-lookup-change{% endif %} lookup" id="{{ id }}_lookup"
                     title="{% translate 'Lookup' %}">
-                        <span class="choose-file"><span class="fa fa-search cms-icon cms-icon-search"></span>{% translate 'Choose File' %}</span>
+                        <span class="choose-file text-input-file"><span class="fa fa-search cms-icon cms-icon-search"></span>Selecione arquivo</span>
```

Comment:
Talvez o uso do `i18n` possa ser necessário, tendo em vista que estamos traduzindo todos os nomes de plugins também! Daí é só dar load em i18n e usar o {% translate 'Choose File' %} mesmo! se caso não tiver pegando, talvez seja o padrão de idiomas do seu navegador!

Reply:
https://docs.djangoproject.com/en/5.0/topics/i18n/translation/#translate-template-tag

Reply:
@miguelzinh3 se for o caso de inserir o i18n como dependência na base do adp acredito que teríamos que abrir um novo PR só para essa inserção. Eu dei uma olhada no projeto e só existe o arquivo .po para tradução ao português apenas no projeto do site do Nossas.

Reply:
@sergiomario @miguelzinh3 o site da AdP não tem tradução, não precisamos focar nessa mudança agora. Mas contextualizando por ser um HTML retirado da biblioteca padrão ele veio com a tradução, mas nosso código é padrão PT-BR e a biblioteca é EN isso muda o valor que temos no default das funções de traduzir.